tools/log-collector/log_collector.go

```diff
@@ -338,11 +347,7 @@ func (l *LogCollector) writeLogs(resourceDir string, obj unstructured.Unstructur
 }
 
 // isSubset checks whether the given namespaces is a subset of all Namespaces in cluster
-func (l *LogCollector) isSubset(second []string) bool {
-	set := make(map[string]string)
-	for _, value := range second {
-		set[value] = value
-	}
+func (l *LogCollector) isSubset(set map[string]string) bool {
```

Comment:
Good for now.. next time, try to use `sets` data type.. there are ready funcs like `IsSuperset` which can used directly.. no need to write custom logic.

Reply:
Alright